projects/storefrontlib/src/cms-structure/outlet/outlet.service.ts

```diff
@@ -9,10 +8,9 @@ export class OutletService<T = TemplateRef<any>> {
   /**
    * @deprecated since 2.1, see #8116
    */
-  constructor();
   // tslint:disable-next-line: unified-signatures
-  constructor(features: FeatureConfigService);
-  constructor(protected features?: FeatureConfigService) {}
+  constructor();
+  constructor() {}
```

Comment:
Deprecation and tslint comments can be removed too, as well as the empty constructor